Integrations/python/deephaven/types.py

```diff
@@ -0,0 +1,25 @@
+# 
+# Copyright (c) 2016 - 2021 Deephaven Data Labs and Patent Pending
+# 
+
+"""
+Functions to return types from jpy for convenience
+"""
+
+import jpy
+
+# Java primitives
+
+Bool = jpy.get_type("boolean")
```

Comment:
I am concerned that we might not be able to do it this simple way.  This would need to be confirmed with Devin.  

If you look in https://github.com/deephaven/deephaven-core/blob/deephaven-types/Integrations/python/deephaven/__init__.py you see: `# NB: this must be defined BEFORE importing .jvm_init or .start_jvm (circular import)`.  I'm concerned that (1) the simple case may not work because of JVM initialization problems or (2) the simple case does work, so we need to clean up the code.

See also cases like this https://github.com/deephaven/deephaven-core/blob/deephaven-types/Integrations/python/deephaven/TableTools/__init__.py, where `_defineSymbols` is used to defer the initialization.

Either way, you need to discuss with @devinrsmith and fire up the code to make sure that things work as expected.